tools/make/golang.mk

```diff
@@ -58,7 +58,7 @@ go.testdata.complete: ## Override test ouputdata
 go.test.coverage: go.test.cel ## Run go unit and integration tests in GitHub Actions
 	@$(LOG_TARGET)
 	KUBEBUILDER_ASSETS="$(shell $(tools/setup-envtest) use $(ENVTEST_K8S_VERSION) -p path)" \
-		go test ./... --tags=integration -race -coverprofile=coverage.xml -covermode=atomic
+		go test ./... --tags=integration -race -coverprofile=coverage.xml -covermode=atomic -p 1
```

Comment:
Sorry for the trouble! Let's leave it! We can change current healthz server port to a non-conflict one.

Reply:
Sure! I will try to address #5149 first, then, and then update this PR post it. Thanks for the help!